components/review/stages/reviewAnswers.tsx

```diff
@@ -0,0 +1,35 @@
+interface Props {
+  questions: string[];
+  answers: string[];
+}
+
+export const ReviewAnswers: React.FC<Props> = ({ questions, answers }) => {
+  return (
+    <div>
+      {questions.map((question, idx) => {
+        return (
+          <div
+            className="flex flex-col gap-5"
+            style={{ alignItems: "flex-start" }}
+          >
+            <h5 className="text-[16px]">{question}</h5>
+            <div
+              className="flex"
+              style={{
+                alignItems: "flex-start",
+                color: "#707070",
+                gap: "9px",
+                padding: "0px 1px",
+              }}
```

Comment:
Can you use the tailwind classes for this?  It will help us to keep our style (colours and padding especially) consistent.